bootstrap

```diff
@@ -112,8 +113,10 @@ extract(Binary) ->
     {ok, Contents}.
 
 request(Url) ->
+    HttpOptions = [{relaxed, true} | get_proxy_auth()],
```

Comment:
I hate that we need to fetch and parse environment variable on every request. There got to be a better way